com.google.Chrome.yaml

```diff
@@ -44,6 +45,30 @@ finish-args:
 modules:
   - libsecret.json
 
+  - name: kerberos
+    subdir: src
+    post-install:
+      - install -Dm644 ../krb5.conf /app/etc/krb5.conf
+    cleanup:
+      - /bin
+      - /sbin
+      - /include
+      - /share/examples
+      - /share/man
+      - '*.pc'
+      - '*.mo'
```

Comment:
>       - '*.mo'

Likely wrong, as libs are using gettext. See `$ grep gettext -R src/lib/`.

Reply:
There are localization files in src/po/ that are built into .mo files and installed.